src/containers/CreatePipelineRun/CreatePipelineRun.test.js

```diff
@@ -12,21 +12,18 @@ limitations under the License.
 */
 
 import React from 'react';
-import { fireEvent, waitFor } from '@testing-library/react';
+import { fireEvent } from '@testing-library/react';
```

Comment:
Most of them are obsolete since they were dealing with updates that are now handled internally by react-query, but some (e.g. the input validation combo tests) will need to be replaced in the next set of changes to the 'create' pages once this phase of work is complete.

These two pages need a pretty significant overhaul as the current input validation code is difficult to maintain. I figured this PR was big enough already so that would come in a follow up change set which would also improve the tests, replacing the large/complex tests with simpler more targeted ones that are easier to maintain.